### ISSUE TITLE
how to genenrate customize dataset npz

### ISSUE THREAD

**Issue by CHAIHEJU** [LTH14/JiT#54]
I use "python -m pytorch_fid --save stats path/to/real-images path path/to/rea_stats.npz"
to generate customize dataset npz but the generted one doesn't works. does anyone done this before?

**Comment by CHAIHEJU** [comment]
problem solved.
use the following code to replace the function "fid_statistics_to_metric" in torch_fidelity/metric_fid.py

```
def fid_statistics_to_metric(stat_1, stat_2, verbose):
    mu1, sigma1 = stat_1["mu"], stat_1["sigma"]
    mu2, sigma2 = stat_2["mu"], stat_2["sigma"]
    def to_numpy(x):
        if isinstance(x, torch.Tensor):
            return x.detach().cpu().numpy()
        return np.array(x)
    mu1 = to_numpy(mu1)
    # ... (Middle omitted) ...
    if sigma2.ndim > 2: sigma2 = sigma2.squeeze()
    #assert mul.ndim == 1 ... 
    #assert sigmal.ndim == 2 ... 
    diff = mu1 - mu2
    tr_covmean = np.sum(np.sqrt(np.linalg.eigvals(sigma1.dot(sigma2))))...
